### ISSUE TITLE
Can't set role of CommandBar to 'toolbar' when you have 'iconOnly' menuitems

### ISSUE THREAD

**Issue by getroyer** [microsoft/fluentui#12353]
### Environment Information

- **Package version(s)**: 7.105.0
- **Browser and OS versions**: Edge, Windows 10

### Describe the issue:

I am working on the Azure Notebooks team. We have an accessibility bug that was assigned to us that involves the CommandBar. Because we are setting iconOnly='True' for the CommandBar items, the CommandBar looks more like a toolbar than a menubar. The bug states that the role of the CommandBar should be 'toolbar' instead of 'menubar'. And the roles of the menuitems should be 'button' instead of 'menuitem'. This is probably because it looks more like a toolbar than a menubar with the iconOnly option set to True.

I was able to set the role of the menuitems to 'button'. But I couldn't set the role of the CommandBar to 'toolbar'.

I found a similar issue that was closed: [https://github.com/OfficeDev/office-ui-fabric-react/issues/12282](https://github.com/OfficeDev/office-ui-fabric-react/issues/12282). It was closed because 'upload button' was not a valid ARIA role. But the difference from that issue is that 'toolbar' is a valid ARIA role.

Here are the MAS References cited in the bug:

[MAS1.3.1](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

[MAS4.1.2](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730})

**Comment by aneeshack4** [comment]
@getroyer Thanks for reaching out to us! I deleted your link to the ADO item (we must be careful and not link internal things on a public platform like Github). Interesting that you were able to override the [menuitems](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx#L166) role with button but not the [menubar role](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.base.tsx#L120)...

**Comment by aneeshack4** [comment]
@getroyer I spoke with the component owner for CommandBar and he pointed out that spec for `toolbar` isn't even complete yet: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_toolbar_role. Toolbar in general seems to be grouped commands, which is not what CommandBar is meant to be used for. So at this moment, we don't believe there's a good reason to allow setting the `toolbar` role on CommandBar. As a result, I'm going to close this but feel free to reach out to us if you have any further questions.
